2019-04-27 20:09:11,946 ERROR: Exception on /user/Ahmed/sendEmail [POST] [in /home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/app/routes.py", line 187, in send_Email
    text_body=render_template('email_/email.txt'),
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: email_/email.txt
